src/Symfony/Component/Serializer/Tests/Normalizer/Features/CallbacksTestTrait.php

```diff
@@ -167,10 +167,7 @@ public function provideNormalizeCallbacks()
             'Collect a property' => [
                 [
                     'bar' => function (array $bars) {
-                        $result = '';
-                        foreach ($bars as $bar) {
-                            $result .= $bar->bar;
-                        }
+                        $result = implode('', $bars);
 
                         return $result;
                     },
```

Comment:
After your simplification, `$result` can be inlined and then the function can be further simplified to an arrow function.

```php
static fn (array $bars) => implode('', $bars)
```